CMakeLists.txt

```diff
@@ -55,6 +54,29 @@ if(
     message(FATAL_ERROR "No cryptography provider is defined.")
 endif()
 
+###########################################################################
+#                                                                         #
+#   Define files for firmware update.                                     #
+#                                                                         #
+#   For more info about firmware update go into                           #
+#   TROPIC01_fw_update_files/README.md                                    #
+#                                                                         #
+#   [1] Define silicon revision, allowed                                  #
```

Comment:
Maybe I'd use directly the name of the macro (sth like `[LT_SILICON_REV]`), instead of numeric references to be more clear.